cmd/convert.go

```diff
@@ -44,6 +45,11 @@ can be converted into a 'konnect' configuration file.`,
 			if err != nil {
 				return fmt.Errorf("converting file: %v", err)
 			}
+			if convertCmdDestinationFormat == "konnect" {
+				cprint.UpdatePrintf("Notice: The `konnect` format type has been deprecated" +
+					"as of v1.12.\nPlease use the `kong-gateway` format with `deck <cmd>` " +
+					"if you would like \nto declaratively manage your Kong gateway config in Konnect.\n")
```

Comment:
@lena-larionova Can we provide a link to a documentation page here as well?

Reply:
Do you want it linking to the CLI reference or an actual guide? 

The CLI ref link would be: https://docs.konghq.com/deck/latest/reference/deck/

The guide link would be: https://docs.konghq.com/konnect/deployment/import/
Note that the guide link won't be live until Helium goes GA. The temp preview site is here: https://konnect--kongdocs.netlify.app/konnect/deployment/import/

Reply:
Let's put in the live link. The window between GA and this going out is not huge and this is only useful once we GA the new feature.

Reply:
Added! Let me know if that's fine now.